compiler/record-minmax/driver/Driver.cpp

```diff
@@ -114,36 +123,20 @@ int entry(const int argc, char **argv)
   std::string input_data_format("h5");
   uint32_t num_threads = 1;
 
-  if (arser["--min_percentile"])
-    min_percentile = arser.get<float>("--min_percentile");
-
-  if (arser["--num_threads"])
-    num_threads = arser.get<int>("--num_threads");
-
+  min_percentile = ::get_values_from<float>(arser, "--min_percentile", 1.0);
```

Comment:
Could you remove the declaration part? Default value is repeated.

If you want to keep that info, it would be better to change function signature.

For example, `update_arg_if_possible<float>(arser, "--min_percentile", min_percentile)` -> update min_percentile if possible, return void